### DISCUSSION TITLE
Move fixed mode image to the centre of the screen

### DISCUSSION THREAD

**Discussion by Groctel** [runelite/runelite#15048] [Ideas]
Add an option in the fixed mode plugin that centres the game window in the middle of the X and Y axis.

**Comment by deathbeam** [comment]
Doesnt it do that already?

**Comment by Groctel** [comment]
It does in the X axis, not in the Y one.
![image](https://user-images.githubusercontent.com/32376781/40013719-a81d1e7c-57ae-11e8-9846-22e92f3fcc04.png)

**Comment by jplsek** [comment]
My interpretation of this:
For stretched mode, 
![image](https://user-images.githubusercontent.com/2388657/40013729-668c75de-577c-11e8-92f7-334525140833.png)
![image](https://user-images.githubusercontent.com/2388657/40013810-a1da2dca-577c-11e8-93f1-151fbb0d1f39.png)
Heck, for regular mode, it could look like this (based on the most recent comment):
![after](https://user-images.githubusercontent.com/2388657/40013826-badbca4a-577c-11e8-9e0c-ad4fbee74ccf.png)

**Comment by Groctel** [comment]
The last picture is exactly what I was asking for :p

**Comment by ffuszthaler** [comment]
are there any updates on this?

**Comment by VossProject** [comment]
I'd love this option too.
